lib/ember-router-generator.js

```diff
@@ -66,11 +64,11 @@ EmberRouterGenerator.prototype.add = function(routeName, options) {
 
 EmberRouterGenerator.prototype._add = function(nameParts, routes, options) {
   options = options || {};
-  var parent   =  nameParts[0];
-  var name     = parent;
+  var parent = nameParts[0];
+  var name = parent;
   var children = nameParts.slice(1);
-  var path     = hasRoute(parent, routes, options.identifier);
-  var route    = path ? path.node : path;
+  var paths     = hasRoute(parent, routes, options.identifier);
+  var route    = paths.length ? paths[paths.length - 1].node : false;
```

Comment:
In this case, even if there are duplicates, we add it to the last one. Does this match the previous behavior of what `hasRoute` returned (first, or last)?